lib/soapy_bing/soap/response/get_ad_groups_by_campaign_id_response.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+require 'active_support/core_ext/array/wrap'
```

Comment:
Also, doesn't `Array(foo)` (Ruby) instead of `Array.wrap(foo)` (ActiveSupport) also work in this case?